opentracing/tracer.py

```diff
@@ -82,3 +83,24 @@ def close(self):
         fut = Future()
         fut.set_result(True)
         return fut
+
+    def implementation_id(self):
+        """Returns the ImplementationID for this OpenTracing implementation.
+
+        :return: An ImplementationID instance that describes this OpenTracing
+            implementation.
+        """
+        return ImplementationID('noop', '1.0.0')
+
+
+# Per collections.namedtuple, the below defines a class called ImplementationID
+# which can be initialized via positional parameters or a kwargs-style
+# initializer.
+#
+#     dapper_impl = ImplementationID('dapper', '0.1.2')
+#     zipkin_impl = ImplementationID(name='zipkin', version='0.3.4')
+#
+ImplementationID = namedtuple('ImplementationID', 'name version')
+
+# The SemVer (http://semver.org/) of the OpenTracing Python API.
+OPENTRACING_PYTHON_SEMVER = '0.9.0'
```

Comment:
(Preface: I do not know python very well, so this might be just a language misunderstanding on my part...)

What I was imagining was given a `Tracer` object, there should be some way to query the supported OpenTracing API version of *that object*.  Having a global does not accomplish quite the same thing.  

At a high-level the problem is the case (impossible in some setups, but not all) were a single logical application has been bound, for one reason or another, to multiple versions of OpenTracing (via use of third-party libraries, nested dependencies, dynamically loaded code, etc.). This case is important because removing the multiple versions is not always something within an individual developer's control (e.g. nested dependencies and third-party code).

In Node.js for example, as far as I know it is fully possible to have two references to two different versions of the same package within a single logical application or service (there will be two distinct copies of the package within the loaded code; to help imagine the real-world scenario think of an application that relies on a framework that relies on third-party extensions that rely on third-party libraries -- and in that complex dependency tree someone has not updated their code to the latest version of OpenTracing but others have). The version check needs to be done on the actual `Tracer` object being used - and can't be a global property as the global property only would reflect the version of *the locally referenced package*, not necessarily *the version that created the Tracer object* in question. 

To clarify this is not Node.js specific, consider a C++ application that dynamically loads a library (i.e. a DLL/.so  plugin of some sort) that, as part of the plugin interface, takes a handle to the active Tracer. It may be interested in knowing the version of that runtime *Tracer object* it has been handed, not the version of OpenTracing the DLL itself was compiled against, to be sure the host application is compatible.  A global check would only check what it had been compiled against.

I certainly wouldn't argue these are going to be a common cases but to achieve the intent, I believe the version info has to be a property of the Tracer object, not a package/module/library-level property.